namerd/docs/storage.md

```diff
@@ -166,6 +166,7 @@ experimental | _required_ | Because this storage is still considered experimenta
 host | `localhost` | The location of the etcd API.
 port | `2379` | The port used to connect to the etcd API.
 pathPrefix | `/namerd/dtabs` | The key path under which dtabs should be stored.
+tls | no tls | Use TLS encryption for connections to Etcd. See [Namer TLS](#namer-tls).
```

Comment:
We're inconsistent about this, but it looks like etcd should not start with an uppercase letter.